api/envoy/type/matcher/string.proto

```diff
@@ -16,7 +16,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // [#protodoc-title: String matcher]
 
 // Specifies the way to match a string.
-// [#next-free-field: 7]
+// [#next-free-field: 8]
```

Comment:
This is still modifying a v2 API. Please revert this.

Reply:
Oh! I thought all the v2 apis are in `api/envoy/api/v2` folder. Reverting.